ocaml/typing/typecore.ml

```diff
@@ -3655,6 +3657,7 @@ let type_pattern_approx env spat ty_expected =
   | _ -> ()
 
 let rec type_function_approx env loc label spato sexp in_function ty_expected =
+  let label = Typetexp.transl_label label in
```

Comment:
Same here. I think perhaps the example I wrote above applies most here. The cases above are for `[%src_pos]` in types, while this one is for terms.